repository/src/main/resources/alfresco/subsystems/Authentication/identity-service/identity-service-authentication-context.xml

```diff
@@ -89,6 +89,12 @@
    </bean>
 
    <bean name="identityServiceConfig" class="org.alfresco.repo.security.authentication.identityservice.IdentityServiceConfig">
+      <property name="issuerUrl">
+         <value>${identity-service.issuer-url:#{null}}</value>

```

Comment:
I'm assuming we need to declare `null` as a default otherwise Spring might complain about not finding the property at all? If it's not the case and the default mapping is unnecessary of course let's get rid of it. But I'm assuming it was done for a reason so it should be good -- just double-checking.

Reply:
I can see there is not difference for that syntax. It seems declaring `null`, it doesn't change anything, the behavior is the same. I got rid of it.

Reply:
I double-checked that. There is a need to declare the default value `null` because not delaring that syntax causes returning string from the value declaration (e.g. `identity-service.issuer-url`) if the property is not set.